pkg/node/statestore.go

```diff
@@ -0,0 +1,25 @@
+package node
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/ethersphere/bee/pkg/logging"
+	"github.com/ethersphere/bee/pkg/statestore/leveldb"
+	"github.com/ethersphere/bee/pkg/statestore/mock"
+	"github.com/ethersphere/bee/pkg/storage"
+)
+
+// InitStateStore will initialze the stateStore with the given data directory.
```

Comment:
The comment is not correct completely. The usage of empty dir should be documented as ti is the main purpose of this function.

Reply:
Good point, updated the comment to clarify the function in more detail.